src/web/resources/report.css

```diff
@@ -8,7 +8,7 @@ figure { margin: 0; }
 
 #large { margin: 2em 0; text-align: center; }
 #large div { margin: 0 1em; display: inline-block; vertical-align: top; }
-#large .number { font-size: 3em; display: block; }
+#large .number { font-size: 2em; display: block; }
```

Comment:
Did you try 2.5? If that doesn't fit, 2 is good, if it does I think we'd prefer bigger text.

Reply:
Got it.